cms/envs/aws.py

```diff
@@ -81,6 +81,9 @@
 CELERY_DEFAULT_QUEUE = DEFAULT_PRIORITY_QUEUE
 CELERY_DEFAULT_ROUTING_KEY = DEFAULT_PRIORITY_QUEUE
 
+# Queue to use for updating grades due to grading policy change
+POLICY_CHANGE_GRADES_ROUTING_KEY = LOW_PRIORITY_QUEUE
```

Comment:
Yes, you would still need them. The "cross the streams" magic from celery.py is a simple `s/cms/lms` text replacement on whatever queue you're using, so the changes in aws/common.py will still "bump it down" from a default queue to a low one.